BismarkUnchained/scenes/Main/Main.tscn

```diff
@@ -1,24 +1,23 @@
 [gd_scene load_steps=7 format=2]
 
 [ext_resource path="res://scenes/Main/Main.gd" type="Script" id=1]
-[ext_resource path="res://scenes/Wall/Wall.tscn" type="PackedScene" id=2]
-[ext_resource path="res://scenes/Player/Player.tscn" type="PackedScene" id=3]
-[ext_resource path="res://scenes/Enemy/Enemy.tscn" type="PackedScene" id=4]
-[ext_resource path="res://scenes/UI/PauseScene.tscn" type="PackedScene" id=5]
-[ext_resource path="res://scenes/UI/InGameUI.tscn" type="PackedScene" id=6]
+[ext_resource path="res://scenes/UI/PauseScene.tscn" type="PackedScene" id=2]
+[ext_resource path="res://scenes/Wall/Wall.tscn" type="PackedScene" id=3]
+[ext_resource path="res://scenes/Player/Player.tscn" type="PackedScene" id=4]
+[ext_resource path="res://scenes/UI/InGameUI.tscn" type="PackedScene" id=5]
+[ext_resource path="res://scenes/WaveHandler/WaveHandler.tscn" type="PackedScene" id=6]
 
 [node name="Main" type="Node2D"]
 script = ExtResource( 1 )
 
-[node name="Wall" parent="." instance=ExtResource( 2 )]
+[node name="Wall" parent="." instance=ExtResource( 3 )]
 
-[node name="Player" parent="." instance=ExtResource( 3 )]
+[node name="Player" parent="." instance=ExtResource( 4 )]
 position = Vector2( 500, 500 )
 scale = Vector2( 1, 1 )
 
-[node name="Enemy" parent="." instance=ExtResource( 4 )]
-position = Vector2( 878.557, 434.408 )
+[node name="PauseScene" parent="." instance=ExtResource( 2 )]
 
-[node name="PauseScene" parent="." instance=ExtResource( 5 )]
+[node name="UI" parent="." instance=ExtResource( 5 )]
 
-[node name="UI" parent="." instance=ExtResource( 6 )]
+[node name="WaveHandler" parent="." instance=ExtResource( 6 )]
```

Comment:
it looks like this is just reordering everything, is this file change necessary?